tests/test_wave_01.py

```diff
@@ -118,12 +118,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 0
     assert len(updated_data["watched"]) is 1
-    
+    assert updated_data["watched"][0]["title"] is MOVIE_TITLE_1
+    assert updated_data["watched"][0]["genre"] is GENRE_1
```

Comment:
This is a much-needed change in our tests but our test asserts should use `==` instead of `is`.`

`==` checks for **value** equality while `is` checks for _reference_ equality (if both references point to the same object in memory). 

`is` is typically used when we check whether a value equates to `None` i.e. "if user_data is None".

Reply:
So in this case the assert should look like:

```suggestion
    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
```